src/main/java/dk/kb/datahandler/kaltura/KalturaDeltaUploadJob.java

```diff
@@ -0,0 +1,304 @@
+        //Next time I recommend a much higher value such as 4096 etc, this is a few seconds of audio.
+        long minimumFileSizeInBytes=700; 
+        String dsStorageUrl = ServiceConfig.getDsStorageUrl();
+        DsStorageClient storageClient = new DsStorageClient(dsStorageUrl);
+                        
+        while (moreSolrRecords) {
+            try {
+                SolrDocumentList docs = fetchSolrRecords(mTimeFromCurrent, 500);
+                if (docs.getNumFound() == 0) {
+                    moreSolrRecords = false; // End loop
+                }
+
+                for (SolrDocument doc : docs) {
+                   String resourceDescription=(String) doc.getFieldValue("resource_description");
+
+                   String title="";//Default
+                   ArrayList<String> titles=(ArrayList<String>) doc.getFieldValue("title"); //multivalue
+                   if (titles.size() >0) {
+                       title = titles.get(0);// take first
+                   }                   
+                   String description=(String) doc.getFieldValue("description");
+                   String fileId=(String) doc.getFieldValue("file_id");
+                   String id=(String) doc.getFieldValue("id");
+                   String originatesFrom= (String) doc.getFieldValue("originates_from");                                       
+                   long mTime= (Long) doc.getFieldValue("internal_storage_mTime");
+                   
+                   mTimeFromCurrent=mTime; //update mTime for next call                   
+                                 
+                 try {                                    
+                   //upload stream
+                   MediaType mediaType = KalturaUtil.getMediaType(resourceDescription);
+                   int flavourParamId = KalturaUtil.getFlavourParamId(mediaType);
+                   String path =KalturaUtil.generateStreamPath(fileId, originatesFrom, resourceDescription);
+                   log.info("validating stream='{}' with title='{}'",path,title);
+                 
+                   String fileError= hasStreamFileError(path, minimumFileSizeInBytes);
+                   if (fileError != null) {
+                       log.warn("File does not exist='{}' or size in bytes less than '{}'. Error='{}'. Id='{}'. Skipping upload", path, minimumFileSizeInBytes,fileError,id);     
+                       updateKalturaIdForRecord(storageClient, fileId, fileError);
+                       continue;                       
+                   }
+                   
+                   // Check file not already in kaltura. 
+                   String kalturaInternalId=getInternalIdKaltura(fileId);
+                   if (kalturaInternalId != null) {
+                       log.warn("Stream allready found in kaltura. FileId='{}'. Setting this kalturaId for record.",kalturaInternalId  );
```

Comment:
In this log FileId gets the value of kalturaInternalId. Maybe update the text FileId to be KalturaId? As FileId and KalturaId are two different IDs used together here